Index.html

```diff
@@ -169,13 +169,17 @@ <h2 class="about">About me</h2>
   </section>
   <section class="page4">
     <p class="pfinal">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-  <form class="cuadrollenado"> 
-    <p class="fill">Full name</p>
-  <img class="fillbar"src="./images/fillbar.svg">
-    <p class="fill">Email address</p> 
-      <img class="fillbar"src="./images/fillbar.svg">
-         <textarea class="comments">Enter text here</textarea>
-</form>
+    
+    <form class="form" method="post"> 
+      <p class="data">Full name</p>
+    <input type="text" name="name" maxlength="30" placeholder="Your name here" required pattern="[A-Za-z0-9]+" required/>
+    <img class="fillbar"src="./images/fillbar.svg">
+    <p class="data">Email adress</p>
+    <input type="email" name="email"  required pattern=".+@gmail+@yahoo+@hotmail\.com" placeholder="example@gmail.com" required/>
+    <img class="fillbar"src="./images/fillbar.svg">
+    <textarea name="textarea" minlength="500" class="data" >Enter text here</textarea>
```

Comment:
- [ ] As stated the project requirement you should make sure that A user cannot submit the form if one or more of the fields is empty, So I recommend you to add the `required` attribute for "text-area" to activate the validation on submit.